web/styles/rendered_markdown.css

```diff
@@ -802,6 +802,22 @@
     }
 }
 
+.message_row.group_mention .message_embed .data-container::after {
+    background: linear-gradient(
+        0deg,
+        var(--color-background-group-mention),
+        transparent 100%
+    );
+}
+
+.message_row.direct_mention .message_embed .data-container::after {
+    background: linear-gradient(
+        0deg,
+        var(--color-background-direct-mention),
+        transparent 100%
+    );
+}
```

Comment:
I think this would make more sense to put in `message_row.css`

Reply:
Yes that makes more sense, I've commited those changes.

Reply:
There was a simpler of doing it and I've commited again, sorry for the confusion. Hope this is enough to merge the PR.